deployments/azuredeploylogs.tf

```diff
@@ -0,0 +1,118 @@
+provider "azurerm" {
```

Comment:
If we include deployment via Terraform, we need to include instructions for this method in our readme